app/views/layouts/_header.html.slim

```diff
@@ -9,20 +9,20 @@ nav.navbar.navbar-inverse.navbar-fixed-top role="navigation"
       a.navbar-brand href=root_path  SpacialDB
     #navbar.collapse.navbar-collapse
       ul.nav.navbar-nav
-        li.active
+        li class=('active' if controller_name == 'home')
           a href="/"  Home
         li
           a href="#about"  About
         li
           a href="#contact"  Contact
-        li
+        li class=('active' if controller_name == 'instances')
           - if user_signed_in?
             a href="/instances"  Instances
```

Comment:
This feature PR hasn't the intention to improve everything. By the way the hardcoding wasn't subject to this PR (as you can see in [1]). So I don't understand the criticism at this point. 

[1] https://github.com/nomadlabs/spacialdb/blame/master/app/views/layouts/_header.html.slim